caom2pipe/execute_composable.py

```diff
@@ -552,66 +420,21 @@ class MetaUpdateClient(CaomExecute):
     def __init__(self, config, storage_name, command_name, cred_param,
                  cadc_data_client, caom_repo_client, observation,
                  meta_visitors, observable):
-        super(MetaUpdateClient, self).__init__(
+        super(MetaUpdate, self).__init__(
             config, mc.TaskType.INGEST, storage_name, command_name, cred_param,
             cadc_data_client, caom_repo_client, meta_visitors, observable)
         self.observation = observation
+        self.logger = logging.getLogger(self.__class__.__name__)
 
     def execute(self, context):
-        self.logger.debug(f'Begin execute for {self.__class__.__name__}')
-        self.logger.debug('the steps:')
-
-        self.logger.debug('Find the file name as stored.')
-        self._cadc_data_info_file_name_client()
-
-        self.logger.debug('create the work space, if it does not exist')
-        self._create_dir()
-
-        self.logger.debug('write the observation to disk for next step')
-        self._write_model(self.observation)
-
-        self.logger.debug('generate the xml, as the main_app will retrieve '
-                          'the headers')
-        self._fits2caom2_cmd_in_out_client()
-
-        self.logger.debug('read the xml from disk')
-        self.observation = self._read_model()
-
-        self.logger.debug('the metadata visitors')
-        self._visit_meta(self.observation)
-
-        self.logger.debug('write the observation to disk for next step')
-        self._write_model(self.observation)
-
-        self.logger.debug('update the observation with xml')
-        self._repo_cmd_update_client(self.observation)
-
-        self.logger.debug('clean up the workspace')
-        self._cleanup()
-
-        self.logger.debug(f'End execute for {self.__class__.__name__}')
-
-
-class MetaUpdateDirect(CaomExecute):
-    """Defines the pipeline step for Collection ingestion of metadata into CAOM.
-    This requires access to only header information.
-
-    This pipeline step will execute a caom2-repo update."""
-
-    def __init__(self, config, storage_name, command_name, cred_param,
-                 cadc_data_client, caom_repo_client, observation,
-                 meta_visitors, observable):
-        super(MetaUpdateDirect, self).__init__(
-            config, mc.TaskType.INGEST, storage_name, command_name, cred_param,
-            cadc_data_client, caom_repo_client, meta_visitors, observable)
-        self.observation = observation
-
-    def execute(self, context):
-        self.logger.debug(f'Begin execute for {self.__class__.__name__}')
+        self.logger.debug('Begin execute')
         self.logger.debug('the steps:')
 
-        self.logger.debug('Find the file name as stored.')
-        self._cadc_data_info_file_name_client()
+        # TODO - what breaks when this isn't here? But it can't be here,
```

Comment:
Is this TODO something that needs attention? I've noticed it in several places.

Reply:
Given that the execute methods now don't care where the data is stored, knowing it's CADC storage name is pointless, so this commented out code will be removed.